scripts/r.import/r.import.py

```diff
@@ -156,8 +166,37 @@ def is_projection_matching(GDALdatasource):
         return False
 
 
+def createvrt(dst, src, nodata=None, region=None, bandlist=None, s_crs=None):
+    # Lazy import
+    from osgeo import gdal
+
+    if not nodata and not region:
+        gs.fatal(_("Either NoData or extent must begiven to create VRT."))
+    params = {"format": "VRT"}
+    if nodata:
+        for nd in nodata.split(","):
+            if not nd.isdigit():
+                gs.fatal(_("Invalid NoData format"))
+        params["noData"] = nodata
+
+    if bandlist:
+        for band in bandlist.split(","):
+            if not band.isdigit():
+                gs.fatal(_("Invalid band number format"))
+        params["bandList"] = bandlist.replace(",", " ").split(" ")
+
+    if region:
+        params["projWinSRS"] = s_crs
+        params["projWin"] = "{w} {n} {e} {s}".format(**region).split(" ")
+
+    indata = gdal.Open(src)
+    vrt = gdal.Translate(dst, indata, options=gdal.TranslateOptions(**params))
+    vrt = None
```

Comment:
Flake8 complains that it is not used. Should it be `del`-eted or the vrt variable not assigned at all at the line above?